website/src/pages/docs/features/introspection.mdx

```diff
@@ -65,7 +65,15 @@ server.listen(4000, () => {
 })
 ```
 
-## Disabling Field Suggestions
+## GraphQL armor
+
+The [`graphql-armor`](https://github.com/Escape-Technologies/graphql-armor) plugin is a security layer that help you protect your GraphQL server from malicious queries.
+It allows you to configure various security features such as character limit or blocking field suggestions.
+For more information about `graphql-armor` features, you can refer to the [documentation](https://escape.tech/graphql-armor/docs/category/plugins).
+
+Here is an example of how to use `graphql-armor` to disable introspection and block field suggestions :
```

Comment:
This page should be feature-oriented, can you please instead revert `Disabling Field Suggestions` to an h2 and put this information instead into an info box?

Reply:
Done.

Need I to create a `<Callout>` box or the `>` works as well ?

Reply:
I recommend using Callout as it will look nicer